internal/rpcapi/cli.go

```diff
@@ -78,10 +78,10 @@ func (c cliCommand) Run(args []string) int {
 			// TODO:
 			//
 			// The following message says that this interface is for Terraform
-			// Cloud only because we're using Terraform Cloud's integration
+			// Cloud only because we're using HCP Terraform's integration
```

Comment:
Flagging this, but a reference to "Terraform Cloud" remains split across lines 80 and 81